### DISCUSSION TITLE
Trouble reading observation file

### DISCUSSION THREAD

**Discussion by DanielMwageni** [dtcenter/METplus#2823] [File I/O] [answered]
Hello, I hope you are having a nice day. I have been been testing some of the METplus components through the METplus practical session guide on the website and everything went well. When I try to replicate the same processes using my own  WRF model dataset and CHIRPS dataset,  most of them work but I am having issues with the grid_stat tool whereby I am getting an error "Trouble reading observation file". As I am still a beginner with MET in general I think the issue might be on my configuration settings, so I am attaching my GridStatConfig_default file. I hope it will help in solving my issue.
[wrf_config_2.txt](https://github.com/user-attachments/files/18094616/wrf_config_2.txt)

**Comment by CPKalb** [comment]
Hi:

Are your CHIRPS observations in netCDF?  If so, could you post of copy of what the header looks like here?  We need that information to help narrow down what may be going on.  If the CHIRPS file is grib, then we would need to see a copy of the output of wgrib/wgrib2.

Alternatively, if it's easier, you could also upload a sample file to our FTP using the [instructions](https://github.com/dtcenter/METplus/discussions/954).

Thanks,
Christina

**Comment by DanielMwageni** [comment, marked as answer]
Thanks for the response but I have already found a way to solve it by modifying the CHIRPS dataset.

**Reply by CPKalb** [reply]
Okay, great.  I'm going to mark this discussion as answered.  If you have future questions, feel free to start a new post.

Christina
